File: OOMTest-Xamarin/OOMTest/MainActivity.cs
using Android.App;
using Android.Widget;
using Android.OS;
using Android.Support.V7.App;
using Android.Support.V4.View;

namespace OOMTest
{
	[Activity(Label = "@string/app_name", MainLauncher = true, Icon = "@mipmap/icon")]
	public class MainActivity : AppCompatActivity
	{
		protected override void OnCreate(Bundle savedInstanceState)
		{
			base.OnCreate(savedInstanceState);
			SetContentView(Resource.Layout.activity_main);

			var viewPager = FindViewById<ViewPager>(Resource.Id.activity_main_view_pager);
			var adapter = new ImageViewPagerAdapter(SupportFragmentManager);
			viewPager.Adapter = adapter;
		}
	}
}

File: OOMTest-Android/app/src/main/java/com/tobishiba/oomtest/MainActivity.java
package com.tobishiba.oomtest;

import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final ViewPager viewPager = (ViewPager) findViewById(R.id.activity_main_view_pager);
        final ImageViewPagerAdapter adapter = new ImageViewPagerAdapter(getSupportFragmentManager());
        viewPager.setAdapter(adapter);
    }
}
